openxla-nvgpu/compiler/src/openxla/compiler/nvgpu/Transforms/CMakeLists.txt

```diff
@@ -17,13 +17,15 @@ iree_cc_library(
     "Passes.h"
     "Passes.h.inc"
   SRCS
-    "Sample.cpp"
-  DEPS
-    ::PassesIncGen
-    MLIRIR
+        "ConvertMHLOToCUDNN.cpp"
```

Comment:
You are right, I failed to run `bazel_to_cmake` so that it would update CMakeLists.txt. Do you know what command I'm supposed to use? And how do you auto-format C++ code?

Anyway, the indentation should be fixed now and I updated the BUILD.bazel file.

Reply:
Nope, I run my cmake files by hand